templates/zerver/emails/email.css

```diff
@@ -364,57 +246,4 @@ a.button:hover {
         border-radius: 0 !important;
         border-right-width: 0 !important;
     }
-
-    table[class="body"] .btn table {
-        width: 100% !important;
-    }
-
-    table[class="body"] .btn a {
-        width: 100% !important;
-    }
-
-    table[class="body"] .img-responsive {
-        height: auto !important;
-        max-width: 100% !important;
-        width: auto !important;
-    }
-}
-
-@media all {
-    .ExternalClass {
-        width: 100%;
-    }
-
-    .ExternalClass,
-    .ExternalClass p,
-    .ExternalClass span,
-    .ExternalClass font,
-    .ExternalClass td,
-    .ExternalClass div {
-        line-height: 100%;
-    }
-
-    /* iOS converts addresses in emails to links automatically */
-    .apple-link a {
-        color: inherit !important;
-        font-family: inherit !important;
-        font-size: inherit !important;
-        font-weight: inherit !important;
-        line-height: inherit !important;
-        text-decoration: none !important;
-    }
```

Comment:
I'm be cautious about this one -- I think it's very possible that iOS email clients insert an `a` tag into the HTML when rendering addresses, as suggested in the (somewhat unclear) comment, and so even though we don't have that syntax, it's actually functional.
```
templates/zerver/emails/email_base_default.html:                                        <span class="apple-link">{{physical_address}}</span>
```